react/index.js

```diff
@@ -182,11 +183,12 @@ class ProductSummary extends Component {
       runtime,
     } = this.props
 
-    const classes = classNames(`${productSummary.container} overflow-hidden br3 w-100 h-100`, {
+    const classes = classNames(`${productSummary.container} overflow-hidden br3 h-100 w-100`, {
       'flex flex-column justify-between center tc': displayMode !== 'inline',
-      [`${productSummary.containerNormal}`]: displayMode === 'normal',
-      [`${productSummary.containerSmall}`]: displayMode === 'small',
-      [`${productSummary.containerInline}`]: displayMode === 'inline',
+      [productSummary.containerNormal]: displayMode === 'normal',
+      [productSummary.containerSmall]: displayMode === 'small',
+      [productSummary.containerInline]: displayMode === 'inline',
+      [productSummary.containerParallel]: displayMode === 'parallel',
```

Comment:
What did you mean with "parallel"?

Reply:
For parallel display like in product kit. I thought of naming "kit" or "productkit" but its sound too specific. I'm open for suggestions :)